### DISCUSSION TITLE
Meaning of "base unit" in a unit

### DISCUSSION THREAD

**Discussion by timkolloch** [TandoorRecipes/recipes#3112] [Q&A] [answered]
What should the base unit mean? 
If I have a unit "Tablespoon" what should be its base unit and what can I do with it?
As a first instinct I would use grams but milliliters may also make sense?

Hopefully someone can tell me how this feature is meant to be used, thanks in advance!

**Comment by smilerz** [comment, marked as answer]
Base unit is the unit that should be used in a recipe.

tablespoon is a unit of volume, you can't use something like grams without knowing the food's density.

**Reply by timkolloch** [reply]
So if I add a base unit of grams to tablespoon and add a conversion, let's say for sugar that states "1 tablespoon of sugar equals 15g". Then for calculation of nutrients in a recipe 3 tablespoons will equal 45 grams of sugar?

If so wouldn't that calculation be also possible without setting a base unit to the unit?

**Reply by smilerz** [reply]
True, what I said is only true for all tablespoon measurements --> grams.  The conversions on Tandoor are done on a food-by-food basis.

**Reply by timkolloch** [reply]
Ah okay, I get that. Thank you very much!
